src/cmd/ship.go

```diff
@@ -106,12 +108,18 @@ func getShipStepList(config shipConfig) (result steps.StepList) {
 	result.Append(&steps.MergeBranchStep{BranchName: branchToMergeInto})
 	result.Append(&steps.EnsureHasShippableChangesStep{BranchName: config.BranchToShip})
 	result.Append(&steps.CheckoutBranchStep{BranchName: branchToMergeInto})
-	result.Append(&steps.SquashMergeBranchStep{BranchName: config.BranchToShip, CommitMessage: commitMessage})
+	canUseDriver := canUseDriver(config.BranchToShip, branchToMergeInto)
```

Comment:
The variable has the same name as the function that is used to calculate its value. Let's not override names like that. What about naming the variable `canShipViaApi`?